Sources/Commands/SwiftPackageTool.swift

```diff
@@ -50,7 +51,7 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
 
         case .initPackage:
             // FIXME: Error handling.
-            let cwd = localFileSystem.currentWorkingDirectory!
+            let cwd = options.packagePath ?? options.chdir ?? localFileSystem.currentWorkingDirectory!
```

Comment:
Could it be worth storing the resolved `packagePage` in a variable in `SwiftTool.init` so we don't have to re-compute here?

Reply:
Yes. Added in a new commit. Thanks!